src/Codeception/Lib/Connector/Yii2/Logger.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Codeception\Lib\Connector\Yii2;
+
+use Codeception\Util\Debug;
+
+class Logger extends \yii\log\Logger
+{
+    public function init()
+    {
+    }
+
+    public function log($message, $level, $category = 'application')
+    {
+        if (!in_array($level, [
+            \yii\log\Logger::LEVEL_INFO,
+            \yii\log\Logger::LEVEL_WARNING,
+            \yii\log\Logger::LEVEL_ERROR,
+        ])) {
+            return;
+        }
+        if (strpos($category, 'yii\db\Command')===0) {
```

Comment:
Could be done via config (`except`) property:

http://www.yiiframework.com/doc-2.0/guide-runtime-logging.html#message-filtering